src/main/java/org/killbill/billing/plugin/avatax/api/AvaTaxTaxCalculator.java

```diff
@@ -226,7 +227,7 @@ private GetTaxRequest toTaxRequest(final String companyCode,
         taxRequest.ExemptionNo = null;
         taxRequest.Discount = BigDecimal.ZERO;
         // Required for VAT
-        taxRequest.BusinessIdentificationNo = null;
+        taxRequest.BusinessIdentificationNo = PluginProperties.findPluginPropertyValue(BUSINESS_IDENTIFICATION_NUMBER, pluginProperties);
```

Comment:
> Could you detail how you run the tests?

To run all the tests, simply execute:

```
mvn clean install -Pmysql
```

Make sure to create a configuration file under `src/test/resources/avatax.properties` (similar format than the one uploaded to Kill Bill).

> However, indeed we could test that the request is still going through.

👍